### ISSUE TITLE
SFTP deploy of binary file (e.g. jpg, ttf, etc.) is fine, but pull of same file corrupts it

### ISSUE THREAD

**Issue by robert914** [mkloubert/vscode-deploy-reloaded#34]
### Description

If I have a binary file (e.g. jpg, ttf, etc.), the file will deploy correctly using SFTP. However, if I try to pull a binary file (e.g. jpg, ttf, etc.) from the server, the local copy is then not the same as the the remote, and is corrupted.

### Actual behavior

Deploy a JPG file (or any binary file), file on remote SFTP is fine.
Pull the same JPG file (or any binary file) from remote SFTPT, the local file is now corrupt.

### Expected behavior

Pull of JPG file (or any binary file) should not get corrupted.

### Steps to reproduce

1. Add a JPG file to workspace.
2. Deploy it using SFTP.
3. Remote JPG is fine.
4. Pull same JPG from SFTP.
5. Local JPG file is now corrupt.

#### Example config

```json
{
    "deploy.reloaded": {
        "packages": [
            {
                "name": "Local",
                "description": "Local source",
                "files": [
                    "**/*",
                ]
            }
        ],
        "targets": [
            {
                "type": "sftp",
                "name": "SRVR",
                "description": "SRVR target",
                "host": "myserver.domain.com",
                "port": 22,
                "user": "myuser",
                "privateKey": "${homeDir}/.ssh/id_rsa_mykey",
                "dir": "/path/to/remote/location"
            }
        ]
    }
}
```

#### Logs

```
Deploying file '/path/to/remote/location/images/banner.jpg' to '/images (EOD)' ... [OK]

Pulling file '/path/to/remote/location/images/banner.jpg' from  '/images (EOD)' ... [OK]
```

#### Screenshot

Not applicable.

### Your environment

- Operating system: MacOS 10.12.6, remote server is Linux RHEL 6.5
- Visual Studio Code version: 1.20.1
- Extension version: 0.53.1

### Additional comments

Deploy and pull of same file should never corrupt it, text or otherwise. Period.

**Comment by zamkevich** [comment]
That look like encoding trouble. For example i'm pull test file.
That's how it's look:
![image](https://user-images.githubusercontent.com/13717428/36933247-754b93cc-1f21-11e8-86c6-0635e1784cdc.png)

**Comment by mkloubert** [comment]
@robert914 @zamkevich 

Thanks a lot!

Right, the default encoding, when downloading a file with [ssh2-sftp-client](https://github.com/jyu213/ssh2-sftp-client), is `utf8` as described [here](https://github.com/jyu213/ssh2-sftp-client#get).

**I fixed and release that with version 0.53.2** by setting encoding to `(null)`.

**Comment by robert914** [comment]
Ok, I installed 0.53.3 and it seems to be working correctly now!
Thanks for the fast response!!
